### PR TITLE
add migration for global notification subscriptions [OSF-5658]

### PR DESCRIPTION
**Pull request by leb2dg**
## Purpose

Subscribe all current users to default notifications.

## Changes

* Added migration for global subscriptions

## Side effects

All users will have global notifications set to instantly unless they already have global notifications.

## Ticket

https://openscience.atlassian.net/browse/OSF-5658

### CODE REVIEW

File: scripts/add_global_subscriptions.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+app = init_app()
+
+
+def add_global_subscriptions():
+
+    notification_type = 'email_transactional'
+    user_events = constants.USER_SUBSCRIPTIONS_AVAILABLE
+
+    for user in models.User.find():
```
Comment (sloria) [inline scripts/add_global_subscriptions.py]: We'll probably want to exclude inactive users (i.e. any users that don't meet the criteria for the `User.is_active` property).

Comment (leb2dg) [inline scripts/add_global_subscriptions.py]: Oh yep. I forgot about unregistered users as well.

**Comment by leb2dg** [comment]
@sloria, updated 🎉

**Comment by sloria** [comment]
Ran locally. Seems to have worked. 🐙

**Pull request merged**